test/lib/test/unit.rb

```diff
@@ -9,6 +9,9 @@
 require 'test/unit/testcase'
 require 'optparse'
 
+require 'simplecov'
+SimpleCov.start
+
```

Comment:
Can we enable this code only when we run our tests on Travis CI?

Reply:
Yes, we can use travis ENVs and restrict the execution based on that. I'll push a change for that.

Reply:
Added the restriction to CI env.

Reply:
Thanks.